buildscripts/make_dependencies.sh

```diff
@@ -26,9 +28,18 @@ else
   pushd $DOWNLOAD_DIR/protobuf-${PROTOBUF_VERSION}
   ./autogen.sh
   # install here so we don't need sudo
-  ./configure --disable-shared --prefix="$INSTALL_DIR"
+  ./configure CFLAGS=-m$ARCH CXXFLAGS=-m$ARCH LDFLAGS=-m$ARCH --disable-shared \
+    --prefix="$INSTALL_DIR"
+  # the same source dir is used for 32 and 64 bit builds, so we need to clean stale data first
+  make clean
```

Comment:
Delete $DOWNLOAD_DIR/protobuf-$PROTOBUF_VERSION before the wget?

Reply:
I wanted to avoid downloading twice, but not feeling too strongly about it.

Reply:
The code _is_ downloading twice, though. Right?

Reply:
Oops!